webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/batch/CheckInventoryUpdate.scala

```diff
@@ -58,6 +62,11 @@ class CheckInventoryUpdate(
   , updateInterval      : Duration
 ) {
 
+  // we need to store the time of last modification we saw so that is someone else update the cache
+  // (for example, a user is using the UI), then even if the cache is upToDate, we now we have to start
+  // a policy generation.
+  val lastUpdate = Ref.make(new DateTime()).runNow
```

Comment:
it may miss the first update with that if the NodeInfoCache is init before the CheckInventoryUpdate. We should use new DateTime(0)

Reply:
I was not sure. Using epoch means that we will *always* trigger a policy generation when the scheduler first run. But we already have one around start time, which mean we will always have two. 
Not sure what the best here.

Reply:
if we don't do that, and there was a change before the scheduler kicked in, it will not apply until a new change is made. I'd rather play safe here